console-src/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I think we can also add 
```
npm-debug.log*
yarn-debug.log*
yarn-error.log*
```
saw it in one of branches

Reply:
True, good idea, I have added it here: 1c4ba19121e44243d42387f75cb4428c83b75cc0